pkg/virt-handler/mount-manager/hotplug-disk/mount_test.go

```diff
@@ -653,46 +539,6 @@ var _ = Describe("HotplugVolume", func() {
 			Expect(unsafepath.UnsafeRelative(res.Raw())).To(Equal(unsafepath.UnsafeAbsolute(expectedPath.Raw())))
 		})
 
-		It("should properly mount and unmount filesystem", func() {
```

Comment:
This can stay or am I missing anything?

Reply:
I was probably deleting too much. I'll readd it

Reply:
Readded